tests/test_register_existing.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+
+import securemailbox
+
+test_fingerprint = "FAC10F0C3D1D49F8F9A82CB553E79F7C92E1CF33"
+
+#incoming_fingerprint = request.json.get("fingerprint")
+#new_mailbox = Mailbox(fingerprint=test_fingerprint)
+#db.session.add(new_mailbox)
+#db.session.commit()
+
+
+def test_register_existing(client):
+    rv = client.post("/register/", json={"fingerprint": test_fingerprint})
+    assert rv.get_json() == {
```

Comment:
This `assert` statement probably shouldn't be here. While it doesn't hurt anything, it causing the test to fail could be misleading. Better yet, I would suggest removing the API call altogether and explicitly add a new mailbox to the database like so:
```python
    new_mailbox = Mailbox(fingerprint=test_fingerprint)
    db.session.add(new_mailbox)
    db.session.commit()
```

Reply:
But the purpose of the test is to successfully add the mailbox then fail to add it again. It expressly works for testing both? I would want to talk about this change before making it